kratos/user_mapper.jsonnet

```diff
@@ -15,9 +8,9 @@ local enshortenUuid(uuid) =
   identity: {
     traits: {
       // it would be better to check if email is verified, but it seems that NBP responds with false even for verified ones
-      email: claims.email,
-      username: enshortenUuid(claims.preferred_username),
-      interest: if 'role' in claims then rolesMap[claims.role] else "",
+      [if "email" in claims then "email" else null]: claims.email,
+      [if "preferred_username" in claims then "username" else null]: enshortenUuid(claims.preferred_username),
+      interest: "",
```

Comment:
I wonder if this should be empty string or null or if it does not matter because either way Kratos redirects to the registration page and asks there.

Reply:
At `interest` we just support empty string, null should lead to an error. The reason of this support is because interest was not required before, but it became required along the way, so there are a lot of users without it.